### ISSUE TITLE
Animation on alert close

### ISSUE THREAD

**Issue by virusakos** [vmware/clarity#404]
**Select one ...**  (check one with "x")
```
[ ] bug
[ ] feature request
[x] enhancement
```

It would be nice to have an animation when a user closes an alert.

### Expected behavior
Animation on alert closing

### Actual behavior
No animation on alert closing

### Reproduction of behavior
Show a closeable alert. Then close it.

### Environment details

* **Angular version:** 2.4.4

* **Clarity version:** 0.8.3

* **OS and version:** macOS 10.12.2

* **Browser:** Tested only on Chrome 55.0.2883.95 (64-bit)

**Comment by reddolan** [comment]
@virusakos did you have an example that we could look at to explain what you mean?

**Comment by virusakos** [comment]
Here is an [example](https://camo.githubusercontent.com/ba10cafaca86dd275323da37dc7da3e627d053bb/68747470733a2f2f6472616761626c7a2e66696c65732e776f726470726573732e636f6d2f323031352f31302f6d6174657269616c64657369676e6469616c6f675f732e676966)

**Comment by reddolan** [comment]
@virusakos thanks for the example

**Comment by colinreedmiller** [comment]
Work on #365 includes a first pass on a motion system including behavior of alerts. This will become a non-issue as soon as the first implementation of motion for components becomes part of a Clarity release since alert close will be one of the first components to be addressed.
Leaving this issue open for now as a reminder.

**Comment by mathisscott** [comment]
This issue has been stale for a while and will ultimately be resolved by what is done in #4206 and potentially #365. I'm closing it as stale and a duplicate. It would be blocked by #4206 in either case.
